### PR TITLE
[Rev 1 Docs] [SRS] [VnVPlan] [VnVReport] Update service and appointments SRS

### PR DESCRIPTION
**Pull request by joyxiao99**
https://github.com/arkinmodi/project-sayyara/issues/256

Update SRS and change vnv due to new appointments requirement
- added a requirement at BE-7 so had to shift all the numbers lol

### CODE REVIEW

File: docs/SRS/SRS.tex [hunk 1]
```diff
@@ -846,18 +854,44 @@ \subsubsection{Appointments}
...
+			            \item The system shall notify the user that the appointment was cancelled
+			            \item The system shall allow the user to exit the cancel appointment process
+		            \end{enumerate}
+	      \end{enumerate}
+
+	\item The user wants to reject an appointment
+	      \begin{enumerate}[VP\arabic*.]
+		      \item Viewpoint: Vehicle Owner
+		            \begin{enumerate}
+			            \item[] N/A
```
Comment (LeonSo7) [inline docs/SRS/SRS.tex:889]: The appointment status shall update and be moved to the rejected/cancelled section

File: docs/SRS/SRS.tex [hunk 2]
```diff
@@ -831,10 +832,17 @@ \subsubsection{Appointments}
...
 			            \item The system shall allow the user to select a scheduled appointment
 			            \item The system shall allow the user to select another available time slot
+			            \item The system shall move the appointment to a requested state
+			            \item The system shall allow the user to cancel and exit the edit appointment process
 		            \end{enumerate}
 		      \item Viewpoint: Auto Repair Shop Owner
 		            \begin{enumerate}
-			            \item[] N/A
+			            \item The system shall allow the user to accept an appointment and move the appointment to a scheduled
+			                  state and notify the user that the appointment was moved to scheduled
```
Comment (arkinmodi) [inline docs/SRS/SRS.tex]: How do we plan to fulfill the "notify the user" part?

File: docs/VnVPlan/VnVPlan.tex [hunk 3]
```diff
@@ -571,14 +626,31 @@ \subsubsection{Appointments}
...
 
 	\item \textbf{FRT-BE7-1}
 
 	      Control: Manual
 
+	      Initial State: Shop Owner/Shop Employee has a service appointment request
+
+	      Input: Selects reject on a requested appointment
+
+	      Output: The appointment booking is rejected and displayed under "rejected appointments" for the
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      Output: The appointment booking is rejected and displayed under ``rejected appointments'' for the
```

File: docs/VnVPlan/VnVPlan.tex [hunk 4]
```diff
@@ -539,12 +540,65 @@ \subsubsection{Appointments}
...
+
+	      Input: Selects an appointment to move to scheduled
+
+	      Output: The appointment is moved to the scheduled appointments tab for the customer and the shop
+	      owner/employee. The user accepting the appointment will receive a notification that the appointment
+	      was accepted.
+
+	      Test Case Derivation: The shop owner/employee user is able accept an appointment request
+
+	      How test will be performed: The tester will go through the user interface to press "accept" on a
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      How test will be performed: The tester will go through the user interface to press ``accept'' on a
```

File: docs/VnVPlan/VnVPlan.tex [hunk 5]
```diff
@@ -539,12 +540,65 @@ \subsubsection{Appointments}
...
+	      Input: Selects an appointment to move to in progress
+
+	      Output: The appointment is moved to in progress appointments tab for the customer and the shop
+	      owner/employee. The user moving the appointment to in progress will receive a notification that the
+	      appointment was moved to in progress
+
+	      Test Case Derivation: The shop owner/employee user is able move a scheduled appointment to in
+	      progress
+
+	      How test will be performed: The tester will go through the user interface to press "in progress" on
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      How test will be performed: The tester will go through the user interface to press ``in progress'' on
```

File: docs/VnVPlan/VnVPlan.tex [hunk 6]
```diff
@@ -539,12 +540,65 @@ \subsubsection{Appointments}
...
+	      Input: Selects an appointment to move to completed
+
+	      Output: The appointment is moved to completed appointments tab for the customer and the shop
+	      owner/employee. The user moving the appointment to completed will receive a notification that the
+	      appointment was moved to completed
+
+	      Test Case Derivation: The shop owner/employee user is able move an in Progress appointment to
+	      completed
+
+	      How test will be performed: The tester will go through the user interface to press "completed" on
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      How test will be performed: The tester will go through the user interface to press ``completed'' on
```

File: docs/VnVReport/VnVReport.tex [hunk 7]
```diff
@@ -372,14 +373,73 @@ \subsection{Appointments}
...
+
+	      Input: Selects an appointment to move to scheduled
+
+	      Output: The appointment is moved to the scheduled appointments tab for the customer and the shop
+	      owner/employee. The user accepting the appointment will receive a notification that the appointment
+	      was accepted.
+
+	      Test Case Derivation: The shop owner/employee user is able accept an appointment request
+
+	      How test will be performed: The tester will go through the user interface to press "accept" on a
```
Comment (arkinmodi) [inline docs/VnVReport/VnVReport.tex]:
```suggestion
	      How test will be performed: The tester will go through the user interface to press ``accept'' on a
```

File: docs/VnVReport/VnVReport.tex [hunk 8]
```diff
@@ -372,14 +373,73 @@ \subsection{Appointments}
...
+	      Input: Selects an appointment to move to in progress
+
+	      Output: The appointment is moved to in progress appointments tab for the customer and the shop
+	      owner/employee. The user moving the appointment to in progress will receive a notification that the
+	      appointment was moved to in progress
+
+	      Test Case Derivation: The shop owner/employee user is able move a scheduled appointment to in
+	      progress
+
+	      How test will be performed: The tester will go through the user interface to press "in progress" on
```
Comment (arkinmodi) [inline docs/VnVReport/VnVReport.tex]:
```suggestion
	      How test will be performed: The tester will go through the user interface to press ``in progress'' on
```

File: docs/VnVReport/VnVReport.tex [hunk 9]
```diff
@@ -372,14 +373,73 @@ \subsection{Appointments}
...
+	      Input: Selects an appointment to move to completed
+
+	      Output: The appointment is moved to completed appointments tab for the customer and the shop
+	      owner/employee. The user moving the appointment to completed will receive a notification that the
+	      appointment was moved to completed
+
+	      Test Case Derivation: The shop owner/employee user is able move an in Progress appointment to
+	      completed
+
+	      How test will be performed: The tester will go through the user interface to press "completed" on
```
Comment (arkinmodi) [inline docs/VnVReport/VnVReport.tex]:
```suggestion
	      How test will be performed: The tester will go through the user interface to press ``completed'' on
```

File: docs/VnVReport/VnVReport.tex [hunk 10]
```diff
@@ -418,6 +480,24 @@ \subsection{Appointments}
 
 	      Control: Manual
 
+	      Initial State: Shop Owner/Shop Employee has a service appointment request
+
+	      Input: Selects reject on a requested appointment
+
+	      Output: The appointment booking is rejected and displayed under "rejected appointments" for the
```
Comment (arkinmodi) [inline docs/VnVReport/VnVReport.tex]:
```suggestion
	      Output: The appointment booking is rejected and displayed under ``rejected appointments'' for the
```

File: docs/VnVPlan/VnVPlan.tex [hunk 11]
```diff
@@ -1342,45 +1414,49 @@ \subsubsection{Legal Requirements}
 \begin{landscape}
 	\subsection{Traceability Between Test Cases and Requirements}
 
-	\begin{longtable}{|l|cccccccccccccccc|}
-		\caption{Traceability Between Functional Test Cases and Functional Requirements, BE1 to BE16}                                                                                                                                                                                                                           \\
+	\begin{longtable}{|l|ccccccccccccccccc|}
+		\caption{Traceability Between Functional Test Cases and Functional Requirements, BE1 to BE17}                                                                                                                                                                                                                                           \\
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex]: Don't update this table to `BE17`. `BE16` is the max size this table can be before it goes off the page.

File: docs/VnVReport/VnVReport.tex [hunk 12]
```diff
@@ -2682,71 +2762,75 @@ \section{Automated Testing}
 \begin{landscape}
 	\section{Traceability Between Test Cases and Requirements}
 
-	\begin{longtable}{|l|cccccccccccccccc|}
-		\caption{Traceability Between Functional Test Cases and Functional Requirements, BE1 to BE16}                                                                                                                                                                                                                           \\
+	\begin{longtable}{|l|ccccccccccccccccc|}
+		\caption{Traceability Between Functional Test Cases and Functional Requirements, BE1 to BE17}                                                                                                                                                                                                                                           \\
```
Comment (arkinmodi) [inline docs/VnVReport/VnVReport.tex]: Don't update this table to `BE17`. `BE16` is the max size this table can be before it goes off the page.

File: docs/VnVPlan/VnVPlan.tex [hunk 13]
```diff
@@ -539,12 +540,65 @@ \subsubsection{Appointments}
...
+
+	      Initial State: Shop Owner/Employee Account has a service appointment requested
+
+	      Input: Selects an appointment to move to scheduled
+
+	      Output: The appointment is moved to the scheduled appointments tab for the customer and the shop
+	      owner/employee. The user accepting the appointment will receive a notification that the appointment
+	      was accepted.
+
+	      Test Case Derivation: The shop owner/employee user is able accept an appointment request
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      Test Case Derivation: The shop owner/employee user is able to accept an appointment request
```

File: docs/VnVPlan/VnVPlan.tex [hunk 14]
```diff
@@ -539,12 +540,65 @@ \subsubsection{Appointments}
...
+
+	      Initial State: Shop Owner/Employee Account has a service appointment scheduled
+
+	      Input: Selects an appointment to move to in progress
+
+	      Output: The appointment is moved to in progress appointments tab for the customer and the shop
+	      owner/employee. The user moving the appointment to in progress will receive a notification that the
+	      appointment was moved to in progress
+
+	      Test Case Derivation: The shop owner/employee user is able move a scheduled appointment to in
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      Test Case Derivation: The shop owner/employee user is able to move a scheduled appointment to in
```

File: docs/VnVPlan/VnVPlan.tex [hunk 15]
```diff
@@ -539,12 +540,65 @@ \subsubsection{Appointments}
...
+
+	      Initial State: Shop Owner/Employee Account has an in progress appointment
+
+	      Input: Selects an appointment to move to completed
+
+	      Output: The appointment is moved to completed appointments tab for the customer and the shop
+	      owner/employee. The user moving the appointment to completed will receive a notification that the
+	      appointment was moved to completed
+
+	      Test Case Derivation: The shop owner/employee user is able move an in Progress appointment to
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      Test Case Derivation: The shop owner/employee user is able to move an in Progress appointment to
```

File: docs/VnVReport/VnVReport.tex [hunk 16]
```diff
@@ -372,14 +373,73 @@ \subsection{Appointments}
...
+
+	      Initial State: Shop Owner/Employee Account has a service appointment requested
+
+	      Input: Selects an appointment to move to scheduled
+
+	      Output: The appointment is moved to the scheduled appointments tab for the customer and the shop
+	      owner/employee. The user accepting the appointment will receive a notification that the appointment
+	      was accepted.
+
+	      Test Case Derivation: The shop owner/employee user is able accept an appointment request
```
Comment (arkinmodi) [inline docs/VnVReport/VnVReport.tex]:
```suggestion
	      Test Case Derivation: The shop owner/employee user is able to accept an appointment request
```

File: docs/VnVReport/VnVReport.tex [hunk 17]
```diff
@@ -372,14 +373,73 @@ \subsection{Appointments}
...
+
+	      Initial State: Shop Owner/Employee Account has a service appointment scheduled
+
+	      Input: Selects an appointment to move to in progress
+
+	      Output: The appointment is moved to in progress appointments tab for the customer and the shop
+	      owner/employee. The user moving the appointment to in progress will receive a notification that the
+	      appointment was moved to in progress
+
+	      Test Case Derivation: The shop owner/employee user is able move a scheduled appointment to in
```
Comment (arkinmodi) [inline docs/VnVReport/VnVReport.tex]:
```suggestion
	      Test Case Derivation: The shop owner/employee user is able to move a scheduled appointment to in
```

File: docs/VnVReport/VnVReport.tex [hunk 18]
```diff
@@ -372,14 +373,73 @@ \subsection{Appointments}
...
+
+	      Initial State: Shop Owner/Employee Account has an in progress appointment
+
+	      Input: Selects an appointment to move to completed
+
+	      Output: The appointment is moved to completed appointments tab for the customer and the shop
+	      owner/employee. The user moving the appointment to completed will receive a notification that the
+	      appointment was moved to completed
+
+	      Test Case Derivation: The shop owner/employee user is able move an in Progress appointment to
```
Comment (arkinmodi) [inline docs/VnVReport/VnVReport.tex]:
```suggestion
	      Test Case Derivation: The shop owner/employee user is able to move an in Progress appointment to
```

**Review by arkinmodi — APPROVED** [review]

**Pull request merged**